### PR TITLE
Load a reduced-to-the-max WebComponent module (HeroBanner only)

### LINKED ISSUE #19: WebC EDS artifact: reduce to max
Goals: 
* Generate one WebC EDS artifact (= JS library) that contains all relevant code for all WebComponents, but does not contain code that is already consumed in EDS:
** Obsolete Themes
** Fonts
** Lit (loaded separately in EDS)
** Design Tokens (CSS Variables)

Non-goals of this ticket: 
* Split the WebC EDS artifact into chunks (Foundation, Extension A, Extension B) - can be evaluated later
* Make the WebC EDS artifact available via a CDN - can be done later

### PR DESCRIPTION
**Pull request by pahupe**
Note: This ticket is mainly being submitted for discussion. It is not complete.

Goals:
* Create a reduced-to-the-max WebComponent module (/libs/eds-wc-bundle1.esm[.m].js). The WebComponent module has been stripped down to only contain the HeroBanner component (following Krisjanis' advice). It still contains some SAP theme CSS and UDex component CSS. Its build process lies outside of this module (locally available only so far)
* Load it in a way that the UI5 components don't switch themes when loading. This requires (so far) a certain loading order (see entries in head.html). Note: The loading of artifacts in head.html is not yet optimized. This is the first attempt to get the theme sap_glow and the WebComponent (here: HeroBanner) loaded and not have the WebComponents loader perform a theme switch (= the reason why the cta-banner-boxes on blog pages turned all blue)

Fix #19

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night
- After: https://19-webc-eds-artifact-reduce-to-max--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -98,19 +98,32 @@ export async function decorateMain(main) {
...
       const head = document.querySelector('head');
       const ui5ThemeScript = document.createElement('script');
       ui5ThemeScript.setAttribute('data-ui5-config', '');
       ui5ThemeScript.setAttribute('type', 'application/json');
       ui5ThemeScript.textContent = `{"theme": "${sapTheme}"}`;
       head.append(ui5ThemeScript);
+      const webComponentsModule = document.createElement('script');
+      webComponentsModule.setAttribute('src', '/libs/dds-wc-bundle.esm.m.js');
+      webComponentsModule.setAttribute('type', 'module');
+      head.append(webComponentsModule);
```
Comment (mhaack) [inline scripts/scripts.js:126]: There is a loadScript helper available.

File: scripts/scripts.js [hunk 2]
```diff
@@ -98,19 +98,32 @@ export async function decorateMain(main) {
...
 
+/**
+ * Add three entries to the head, in this order:
+ * <!-- Load SAP theme (here: sap_glow) -->
+ * <link rel="stylesheet" href="/themes/sap_glow/css_variables.css">
+ * <!-- Announce theme to UI5 components (before module is loaded) -->
+ * <script data-ui5-config type="application/json">{"theme": "sap_glow"}</script>
+ * <!-- Load SAP Digital Design System Web Components Module -->
+ * <script src="/libs/dds-wc-bundle.esm.m.js" type="module"></script>
+ * @returns {Promise<void>}
```
Comment (mhaack) [inline scripts/scripts.js:109]: This is waste of bytes send to the user, the code below is clear and simple.

File: scripts/scripts.js [hunk 3]
```diff
@@ -98,19 +98,32 @@ export async function decorateMain(main) {
...
+ */
 async function loadSAPTheme() {
```
Comment (mhaack) [inline scripts/scripts.js:111]: Nitpicking, I know :-), if this method is going to load WebComponent stuff as well, we should rename it.

File: scripts/scripts.js (hunk 1 above)
Comment (pahupe) [reply scripts/scripts.js]: Incorporated.

File: scripts/scripts.js (hunk 2 above)
Comment (pahupe) [reply scripts/scripts.js]: Removed.

File: scripts/scripts.js (hunk 3 above)
Comment (pahupe) [reply scripts/scripts.js]: Renamed.

**Review by karlpauls — APPROVED** [review]

**Pull request merged**